kernel/mount.c

```diff
@@ -299,6 +310,114 @@ static const char* _find_arg(const char* args[], const char* name)
 }
 #endif /* MYST_ENABLE_EXT2FS */
 
+#ifdef MYST_ENABLE_HOSTFS
+static bool _find_mount_source(const char* source, char path[PATH_MAX])
```

Comment:
I think I would consider returning a const pointer to the path rather than making a copy.

Reply:
Good point!